### PR TITLE
Add a Makefile

### PR DESCRIPTION
**Pull request by Nakaner**
This should model all dependencies for a smooth developing experience without using Kosmtik.

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -0,0 +1,18 @@
+CARTO = carto
+
+all: standard maxspeed signals electrification
```
Comment (DerDakon) [inline Makefile]: a rule for electrification is missing

File: Makefile [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+
+standard: project.xml
+
+maxspeed: maxspeed.xml
+
+signals: signals.xml
+
+project.xml: project.mml fonts.mss standard_labels.mss standard.mss standard_symbols.mss
+	$(CARTO) $< > $@
```
Comment (DerDakon) [inline Makefile]: Even if this is a rather trivial rule at the moment, I wonder if we should go for a general xml rule like ```%.xml:``` and just list the extra dependencies for the xml files. The standard rule can then depend on e.g. fonts.mss already.

File: Makefile [hunk 3]
```diff
@@ -0,0 +1,18 @@
+CARTO = carto
+
+all: standard maxspeed signals electrification
+
+standard: project.xml
```
Comment (DerDakon) [inline Makefile]: Why this mismatch in naming?

File: util/get_mml_dependencies.py [hunk 4]
```diff
@@ -0,0 +1,39 @@
...
+import argparse
+import os
+import os.path
+import sys
+import yaml
+
+parser = argparse.ArgumentParser(
+    description="""
+Print the names of the .mss files used by a .mml file (CartoCSS style, YAML
+format) and the name of the .mml file itself as a list of ependencies of a
```
Comment (DerDakon) [inline util/get_mml_dependencies.py]: "ependencies" ;)

File: util/get_mml_dependencies.py [hunk 5]
```diff
@@ -0,0 +1,39 @@
...
+)
+parser.add_argument(
+    "input_file",
+    type=argparse.FileType("br"),
+    help="Input file"
+)
+args = parser.parse_args()
+
+# Check the current working directory is correct.
+if os.path.abspath(os.path.dirname(args.input_file.name)) != os.getcwd():
```
Comment (DerDakon) [inline util/get_mml_dependencies.py]: Why is this important? Or just cd into the correct directory?

File: .gitignore [hunk 6]
```diff
@@ -1,6 +1,8 @@
 *.swp
 data/
-project.xml
+standard.xml
 maxspeed.xml
 signals.xml
 *.mapcss
+Makefile
```
Comment (jonsger) [inline .gitignore]: Do we really want to ignore the Makefile? Shouldn't it be tracked by git?

Comment (DerDakon) [reply .gitignore]: yes, this line must go away

File: util/get_mml_dependencies.py (hunk 5 above)
Comment (Nakaner) [reply util/get_mml_dependencies.py]: I removed it and it works from anywhere now.

**Comment by Nakaner** [comment]
I rebased on current master and think that it is ready for a final review.

**Pull request merged**